bcycle-backend/Controllers/TripsController.cs

```diff
@@ -24,30 +25,27 @@ public TripsController(TripService tripService)
 
         // GET /api/trips
         [HttpGet]
-        public async Task<ActionResult<ResultContainer<IEnumerable<TripDto>>>> Get()
+        public ActionResult<ResultContainer<IEnumerable<TripResponse>>> Get()
         {
-            var trips = await _tripService.GetAll(User.GetId()).ToListAsync();
-            return new ResultContainer<IEnumerable<TripDto>>(trips);
+            var trips = _tripService.GetAll(User.GetId()).Select(t => t.AsResponse()).ToList();
```

Comment:
Why is this not done asynchronously anymore?

Reply:
Not sure. Fixed.

Reply:
I remember now. Didn't want service to expose IQueryable interface as it is db specific. But now when i think about it, it doesn't matter.